corehq/apps/mocha/templates/mocha/base.html

```diff
@@ -0,0 +1,34 @@
+{% load hq_shared_tags %}
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+    <title>CommCareHQ Test</title>
+    <script type="text/javascript" src="{% new_static 'jquery/dist/jquery.min.js' %}"></script>
+    <!-- Mocha and test-only dependencies -->
+    <link href="{% new_static 'mocha/mocha.css' %}" type="text/css" rel="stylesheet"></link>
+    <script src="{% new_static 'mocha/mocha.js' %}"></script>
+    <script src="{% new_static 'chai/chai.js' %}"></script>
+
+    <!-- Core Libraries -->
+    {% block core_libraries %}{% endblock %}
+
+    <!-- App specific dependencies -->
+    {% block dependencies %}{% endblock %}
+
+  </head>
+  <body>
+      <div id="mocha"></div>
+    <script>
+      mocha.setup('bdd')
+      window.assert = chai.assert
```

Comment:
this is what we're used to, but i'd also make the argument for the BDD style asserts. for one, the rest of the testing framework is in BDD syntax (`describe`, `it`), and two, it's actually kind of nice that it's clear which value you expect to what. for example

```
expect(myValue).toBe(someOtherValue)
// as opposed to
assert.equal(myValue, someOtherValue)
```
I think in the first one it's actually more clear that the `someOtherValue` is the source of truth. not gonna push hard on this because we're already used to asserts with python, but it is a paradigm that has some benefits